src/Bank.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Created by darionmoore on 11/29/16.
+ */
+public class Bank {
+    private int bankMoney ;
+    private String hotels;
+
+    public Bank(int bankMoney, String hotels) {
+        this.bankMoney = bankMoney;
+        this.hotels = hotels;
+
+    }
+
+    public int getBankMoney() {
+        return this.bankMoney;
+
+
+    }
+
+    public String getHotels() {
+        return this.hotels;
+
+    }
+
+    public void setBankMoney(int bankMoney) {
```

Comment:
Being able to set the bank money is probably too loose of a way to do this.  If you think about this flow the calling code will have to `getMoney()` then do the balance calculation before calling `setMoney`.  You would probably have cleaner code by making the bank add and remove money similarly to the `Player` class.